e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -69,12 +69,17 @@ Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage
   await addHearingBookingDetailsEventPage.enterHearingDetails(hearingDetails[1]);
   await addHearingBookingDetailsEventPage.enterJudge(hearingDetails[1].judgeAndLegalAdvisor);
   await addHearingBookingDetailsEventPage.enterLegalAdvisor(hearingDetails[1].judgeAndLegalAdvisor.legalAdvisorName);
-  await I.completeEvent('Save and continue', {summary: 'summary', description: 'description'});
+  await I.retryUntilExists(() => I.click('Continue'), '#newHearingSelector_newHearingSelector');
+  addHearingBookingDetailsEventPage.sendNoticeOfHearing(hearingDetails[0].sendNoticeOfHearing);
+  addHearingBookingDetailsEventPage.sendNoticeOfHearing(hearingDetails[1].sendNoticeOfHearing, 1);
+  await I.completeEvent('Save and continue');
```

Comment:
Yeah I agree there's no need to test this flow twice. The e2e test was already there, I had only updated it to make sure it continues to work. Currently we need this event because the mock data does not include hearingDetails data. 

Perhaps we could update this file so fixture does include hearings and test just ensures user can read from tabs.

Reply:
Caught up with Adam and agreed to keep this in for now.